vpype/hpgl_devices.toml

```diff
@@ -40,3 +40,38 @@ y_range = [0, 10365]
 y_axis_up = true
 origin_location = ["15mm", "287mm"]
 set_ps = 0
+
+[device.calcomp_designmate]
+name = "designmate"
+plotter_unit_length = "0.02488mm"
+pen_count = 8
+
+[[device.calcomp_designmate.paper]]
+name = "a4"     # Will always be loaded in Landscape oriebtaion in the Designmate
+aka_names = ["A4"]
+paper_size = ["297mm", "210mm"]
+x_range = [0, 11040]
+y_range = [0, 7721]
+y_axis_up = true
+origin_location = ["12mm", "198mm"]
+set_ps = 0
+
+[[device.calcomp_designmate.paper]]
+name = "a3"  # As loaded in portrait orientaion in the plotter. Does not affect drawing orientaion, only page margins
+aka_names = ["A3"]
+paper_size = ["420mm", "297mm"]
+x_range = [0, 15207]
+y_range = [0, 10853]
+y_axis_up = true
+origin_location = ["12mm", "272mm"]
```

Comment:
This needs adjustment as well. I'm expecting something along the lines of ["185mm", "12mm"], values to be checked (I don't have all the details in mind anymore).

Reply:
checked